scripts/check_configs.ts

```diff
@@ -596,6 +675,16 @@ async function main() {
         outcomeMap[network] = false;
       }
 
+      // query all bridges
+      const bridgesMap: BridgeMap | undefined = sdk?.token?.bridges
+      let bridgesArr: string[] = []
+
+      if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
+        const polynetworkBridges = bridgesMap.polynetwork.map((bridge) => bridge.bridgeAddresses).flat()
+        const ibcBridges = bridgesMap.ibc.map((bridge) => bridge.bridgeAddresses).flat()
+        bridgesArr = polynetworkBridges.concat(ibcBridges)
```

Comment:
Pls don't forget to include the axelar bridge ids (i.e. `bridge.connectionId`).
![image](https://github.com/user-attachments/assets/fc2f3045-9820-48d4-a0be-72638419a66e)